src/domain/proposal.ts

```diff
@@ -65,6 +65,19 @@ export function getProposalIPFSData(proposal: Proposal): Proposal {
         if (descJson.toObject().get('url') != null) {
           proposal.url = descJson.toObject().get('url').toString();
         }
+        let tags: string[] = [];
+        let tagsData =  descJson.toObject().get('tags');
+        if (tagsData != null && tagsData.kind === JSONValueKind.ARRAY) {
+          let tagsObjects = tagsData.toArray();
+          for (let i = 0; i < tagsObjects.length; i++) {
+            tags.push(tagsObjects[i].toString());
+            let tagEnt = new Tag(tagsObjects[i].toString());
+            if (Tag.load(tagsObjects[i].toString()) == null) {
```

Comment:
this check should move to line 73 ?

Reply:
No... we want the tag in the list of the proposal anyway